faunadb-java/src/test/java/com/faunadb/client/ClientSpec.java

```diff
@@ -22,9 +22,11 @@
 import static java.util.Arrays.asList;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.Flow;
+import java.util.concurrent.*;
```

Comment:
Let's not use "*" imports

Reply:
@shestakovg
1. Could you fix the imports, so they don't contain `*` imports?
2. Could you also call `Arrays.asList` directly in the code instead of using static import?